src/Compilers/CSharp/Test/Semantic/Semantics/IteratorTests.cs

```diff
@@ -407,5 +407,27 @@ protected override IEnumerable<int> M()
             var comp = CompileAndVerify(source, expectedOutput: "0,1,2,3", options: TestOptions.DebugExe);
             comp.Compilation.VerifyDiagnostics();
         }
-   }
+
+        [Fact]
+        [WorkItem(261047, "https://devdiv.visualstudio.com/DevDiv/_workitems?id=261047&_a=edit")]
+        public void MissingExpression()
```

Comment:
Yes, due to recent changes in scoping rules for Out Variables and Pattern Variables, this issue is no longer reproducible. Adding a test to close the test gap.